mps_youtube/main.py

```diff
@@ -3280,6 +3287,22 @@ def yt_url(url, print_title=0):
     if print_title:
         xprint(v.title)
 
+def yt_url_file(file_name):
+    """ Access a list of urls in a text file """
+
+    #Open and read the file
+    try:
+        with open(file_name, "r") as fo:
+            output = ','.join([line.strip('\n') for line in fo])
```

Comment:
@bendem Oh, I didn't see that comment before merging, but I agree: cccbf0894172a738732715c1f8f876b0992e49f5